medusa/tv/episode.py

```diff
@@ -96,12 +82,12 @@
 except ImportError:
     import xml.etree.ElementTree as ETree
 
-shutil.copyfile = shutil_custom.copyfile_custom
+log = BraceAdapter(logging.getLogger(__name__))
+log.logger.addHandler(logging.NullHandler())
 
+shutil.copyfile = shutil_custom.copyfile_custom
```

Comment:
This shouldn't be here. I don't know how it came back since all these `hacks` are initialized when we start medusa: https://github.com/pymedusa/Medusa/blob/develop/medusa/init/__init__.py#L24

I moved all to a single place to keep it clean and easy to maintain/remove